egs2/TEMPLATE/enh1/enh.sh

```diff
@@ -58,15 +58,15 @@ enh_tag=    # Suffix to the result dir for enhancement model training.
 enh_config= # Config for enhancement model training.
 enh_args=   # Arguments for enhancement model training, e.g., "--max_epoch 10".
             # Note that it will overwrite args in enhancement config.
-spk_num=2   # Number of speakers
-dynamic_mixing=false # Flag for dynamic mixing in speech separation task. 
+ref_num=2   # Number of references (similar to speakers)
+inf_num=    # Number of inferences output by the model
+            # If not specified, it will be the same as ref_num. If specified, it will be overwritten.
```

Comment:
Note that all `--spk_num` arguments specified in `enh1` recipes should be replaced after this change. (For example, `egs2/chime4/enh1/run.sh`.)

Reply:
Also it is better to clarify the meaning of `ref_num` in different cases.
For example:
  * when MixIT-based training strategy is used, `ref_num` means the number of reference signals, while each reference signal may contain more than one speaker.
  * otherwise, `ref_num` is equivalent to the number of speakers in normal speech enhancement/separation tasks.